sandstone/custom-skin/src/components/ColorFields/ColorFields.js

```diff
@@ -4,15 +4,58 @@ import PropTypes from 'prop-types';
 import ColorField from '../ColorField/ColorField';
 import NameField from '../NameField/NameField';
 
+/**
+ * A component that contains all the input fields of the custom-skin application.
+ * It contains a NameField for the skin and one ColorField for each color that can be customized
+ */
 const ColorFields = kind({
 	name: 'ColorFields',
 
 	propTypes:{
-		auto:PropTypes.bool,
-		colors:PropTypes.array,
-		name: PropTypes.string,
-		onChangeInput:PropTypes.func,
-		propNames:PropTypes.array
+		/**
+		 * Variable that indicates the auto mode
+		 *
+		 * @type {Boolean}
+		 * @required
+		 * @public
+		 */
+		auto:PropTypes.bool.isRequired,
```

Comment:
Have you run lint this project?
```suggestion
		auto: PropTypes.bool.isRequired,
```

Reply:
I see lots of this..

Reply:
Seems that enact lint -s doesn't catch this :( Thanks for noticing
Lint was ran all the time